app/helpers/application_helper.rb

```diff
@@ -491,4 +491,22 @@ def user_change_description(version)
       end
     end
   end
+
+  def canceled_replacement_event_label(event, event_schedule, *label_classes)
+    if event.state == 'canceled' || event.state == 'withdrawn'
```

Comment:
That's a bit too random. We don't really want this for any cancelled/withdrawn proposal, just the ones that are scheduled.